.gitignore

```diff
@@ -0,0 +1,55 @@
+# Compiled source #
```

Comment:
Do we need all these filetypes listed as ignored? Could you restrict the list to just what we need?